salesagility/SuiteCRM
# Feature codeception module file

Task: SCRM-661

This change includes #4044 and #4049  - Please review and merge this in first.

## Description



This change adds acceptance tests for the file sugar object. 

## Motivation and Context

Tests company module:
- create file module in module builder
- view file module
- create record with file module and upload an automatically generated test file
- filter and select record
- edit record
- duplicate record
- delete record


## How To Test This

* See [tests/README.md](https://github.com/daniel-samson/SuiteCRM/blob/feature_codeception_basic_module/tests/README.md) and configure your environment appropriately
* Run `composer install`
* Run `codecept run install`
* Run `codecept run tests/acceptance/Core/FileModuleCest.php`

## Types of changes


### Final checklist
